my_list.rb

```diff
@@ -0,0 +1,22 @@
+require_relative './my_enumerable'
+
+class MyList
+  def initialize(list)
+    @list = list
+  end
+
+  include MyEnumerable
+
+  def each(&block)
+    @list.each(&block)
+  end
```

Comment:
- [ ] `Each` method should `yield` all members of `@list` in succession. you can check how to use [yield](https://www.educba.com/yield-in-ruby/)